scripts/pylint_extensions_test.py

```diff
@@ -1469,3 +1469,96 @@ def test_well_placed_newline(self):
 
         with self.checker_test_object.assertNoMessages():
             temp_file.close()
+
+
+class DivisionOperatorCheckerTests(unittest.TestCase):
+
+    def test_division_operator(self):
+        checker_test_object = testutils.CheckerTestCase()
+        checker_test_object.CHECKER_CLASS = (
+            pylint_extensions.DivisionOperatorChecker)
+        checker_test_object.setup_method()
+        node_division_operator = astroid.scoped_nodes.Module(
+            name='test',
+            doc='Custom test')
+        temp_file = tempfile.NamedTemporaryFile()
+        filename = temp_file.name
+
+        with python_utils.open_file(filename, 'w') as tmp:
```

Comment:
Can you split each of these test cases up? The reason we want to split it up is so that it's easier to spot what you are testing. Conversely, in the case of failure, we know what caused it exactly.

Reply:
Done.